packages/stores/src/account/base.ts

```diff
@@ -91,6 +93,14 @@ import { WalletConnectionInProgressError } from "./wallet-errors";
 
 export const GasMultiplier = 1.5;
 
+// The number of heights from current before transaction times out.
+// 10 heights * 5 second block time = 50 seconds before transaction
+// timeout and mempool eviction.
+const timeoutHeightOffset: bigint = BigInt(10);
```

Comment:
Will move to Vercel config: https://github.com/osmosis-labs/osmosis-frontend/issues/2740